atomics/T1497.001/T1497.001.yaml

```diff
@@ -0,0 +1,39 @@
+---
+attack_technique: T1497.001
+display_name: 'Virtualization/Sandbox Evasion: System Checks'
+atomic_tests:
+- name: Detect Virtualization Environment (Linux)
+  description: |
+    systemd-detect-virt detects execution in a virtualized environment.
+    At boot, dmesg stores a log if a hypervisor is detected.
+  supported_platforms:
+    - linux
+  executor:
+    name: sh
+    elevation_required: false 
+    command: | 
+      if (systemd-detect-virt || dmesg | grep -i "hypervisor detected") then echo "Virtualization Environment detected"; fi;  
```

Comment:
Id prefer the fallback here being similar to the `macos` method, using `dmidecode` to detect manufacturer. The kernel ring buffer can roll, be turned off, or cleared.

Example to detect VMWare
`sudo dmidecode | egrep -i 'manufacturer|product|vendor' | grep -i VMWare`

Reply:
Sure @rc-grey . Thanks. Changed it.

Reply:
Thanks for the contribution!